src/commands/flow/scan.ts

```diff
@@ -83,27 +97,36 @@ export default class scan extends SfdxCommand {
     } else {
       flowFiles = FindFlows(".");
     }
+    this.ux.startSpinner(`Found ${flowFiles.length} flows to analyze`);
 
     // Perform scan
     const parsedFlows: Flow[] = await ParseFlows(flowFiles);
+    this.ux.startSpinner(`Parsed ${flowFiles.length} flows, processing analysis`);
     let scanResults: ScanResult[];
     if (this.userConfig && Object.keys(this.userConfig).length > 0) {
       scanResults = core.scan(parsedFlows, this.userConfig);
     } else {
       scanResults = core.scan(parsedFlows);
     }
 
+console.log(JSON.stringify(scanResults));
```

Comment:
Rm console log please

Reply:
Work in progress, I wouldn't dare to let it here haha ^^